feature-libs/customer-ticketing/core/customer-ticketing-constants.ts

```diff
@@ -1 +1,2 @@
+export const MAX_INPUT_CHARACTERS_FOR_MESSAGE = 5000;
```

Comment:
as mentioned by someone in a comment, let's remove this constant and set it to 5000

Reply:
therefore removing the needing to have different limits https://github.com/SAP/spartacus/pull/16063/files#diff-a9e6c1c63dd6600ad490edb4ecfb7b650dffc06530b5abe6d7ed1526ed77a415R18